src/main/kotlin/string/Calculator.kt

```diff
@@ -0,0 +1,58 @@
+package string
+
+class Calculator() {
+    fun sum(input: String?): Int {
+        if (input.isNullOrBlank()) {
+            return 0
+        }
+
+        return add(parse(input))
+    }
+
+    private fun add(inputs: List<String>): Int {
+        var result = 0
+
+        for (input in inputs) {
+            result += input.toInt()
+        }
+
+        return result
+    }
```

Comment:
코틀린 확장 함수를 이용하면 아래 코드처럼 한 줄로 줄여볼 수 있어요!
```suggestion
    private fun add(inputs: List<String>): Int {
        return inputs.sumOf { it.toInt() }
    }
```

Reply:
오 이런게 있었군요..!! 감사합니다.